maintainers/maintainer-list.nix

```diff
@@ -10098,6 +10098,12 @@
     githubId = 10473184;
     name = "Jia Xiaodong";
   };
+  polarmutex = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.